File: BlueJ/DiagramaDeClases/Mascota.java
public class Mascota
{
    int intCodigo;
    String strNombre;
    String strEspecie;
    String strRaza;
    int intEdad;
    Propietario objPropietario;

    public Mascota(int intCodigo, String strNombre,
                   String strEspecie, String strRaza,
                   int intEdad, Propietario objPropietario)
    {
        this.intCodigo = intCodigo;
        this.strNombre = strNombre;
        this.strEspecie = strEspecie;
        this.strRaza = strRaza;
        this.intEdad = intEdad;
        this.objPropietario = objPropietario;
    }

    public void mostrarMascota()
    {
        System.out.println("Codigo: " + intCodigo);
        System.out.println("Nombre: " + strNombre);
        System.out.println("Especie: " + strEspecie);
        System.out.println("Raza: " + strRaza);
        System.out.println("Edad: " + intEdad);
        System.out.println("Propietario: " + objPropietario.strNombre);
    }
}

File: Codigo/SistemaVeterinario/logica/Mascota.cs
namespace SistemaVeterinario.logica
{
    public class Mascota
    {
        public int intCodigo { get; set; }
        public string strNombre { get; set; }
        public string strEspecie { get; set; }
        public string strRaza { get; set; }
        public int intEdad { get; set; }
        public Propietario objPropietario { get; set; }

        public Mascota(int intCodigo, string strNombre, string strEspecie,
                       string strRaza, int intEdad, Propietario objPropietario)
        {
            this.intCodigo = intCodigo;
            this.strNombre = strNombre;
            this.strEspecie = strEspecie;
            this.strRaza = strRaza;
            this.intEdad = intEdad;
            this.objPropietario = objPropietario;
        }

        public void mostrarMascota()
        {
            Console.WriteLine("Codigo: " + intCodigo);
            Console.WriteLine("Nombre: " + strNombre);
            Console.WriteLine("Especie: " + strEspecie);
            Console.WriteLine("Raza: " + strRaza);
            Console.WriteLine("Edad: " + intEdad);
            Console.WriteLine("Propietario: " + objPropietario.strNombre);
        }
    }
}